src/components/form/components/checkbox.js

```diff
@@ -21,7 +21,7 @@ const Checkbox = ({
       className={classnames('checkbox', className)}
       style={style}
     >
-      <Element type="checkbox" disabled={disabled} {...props} /> {children}
+      <Element type="checkbox" disabled={disabled} checked={checked} {...props} /> {children}
```

Comment:
Thanks. Instead of adding this here nearby could be better removing it for the desructuring.

Reply:
Fixed in 3a40d91ec559277bd703a8c83bf5fc8e83957fb5